### ISSUE TITLE
Race links

### ISSUE THREAD

**Issue by jacoboyen** [jacoboyen/iowacx#1]
Per suggestion, we should link each race to the USAC site.

**Comment by scottolsen** [comment]
Just noticed some races have their own site and/or fb page. Thoughts on something like this? Links to fb page, usac and flyer.

<img width="406" alt="iowa_cyclocross_series" src="https://user-images.githubusercontent.com/113754/29617936-2c47f0bc-87dc-11e7-925a-a6f564e6e957.png">

**Comment by jacoboyen** [comment]
That totally makes sense. Think it makes sense to lead with the USAC icon since that is where the registration and flyer goodies are?

Looks like I didn't style `a:link, a:visited` in the stylesheet too. Whoops.

**Comment by scottolsen** [comment]
Agree, fb can probably go last. What about races that have 2 days? List icons for both or just first one?

**Comment by jacoboyen** [comment]
I'm a fan of consistency so I'd just use the icons for both days even though it goes to the same place. It might look a little awkward if we only do it on the first day.
